src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -280,3 +305,53 @@ const optionsValidate: ValidationFnProps = (selected, optionsValues) => {
   // XXX TODO
   return true;
 };
+
+function SubmitConfirmation(
+  actionName: string,
+  unitCount: number,
+  unitList: string[],
+  confirmFunction: () => void,
+  cancelFunction: () => void
+): JSX.Element {
+  const unitNames = unitList.reduce((acc, unitName) => {
+    return `${acc}, ${unitName.split("/")[1]}`;
+  });
+  return (
+    <ConfirmationModal
+      buttonRow={
+        <div>
+          <button
+            className="p-button--neutral"
+            key="cancel"
+            onClick={cancelFunction}
+          >
+            Cancel
+          </button>
+          <button
+            className="p-button--positive"
+            key="save"
+            onClick={confirmFunction}
+          >
+            Confirm
+          </button>
+        </div>
+      }
+    >
+      <div className="actions-panel__confirmation-modal">
+        <h4>Run {actionName}?</h4>
+        <div className="actions-panel__confirmation-modal-info-group">
```

Comment:
Then this becomes `p-confirmation-modal__info-group`

Reply:
done